src/nft/components/collection/Sweep.tsx

```diff
@@ -14,11 +14,15 @@ import { ThemedText } from 'theme'
 const SweepContainer = styled.div`
   display: flex;
   gap: 60px;
-  margin-top: 20px;
+  margin: 12px 16px 0 16px;
   padding: 16px;
   border-radius: 12px;
-  background-color: ${({ theme }) => theme.backgroundModule};
+  background-color: ${({ theme }) => theme.backgroundSurface};
   justify-content: space-between;
+  background: linear-gradient(${({ theme }) => theme.backgroundSurface}, ${({ theme }) => theme.backgroundSurface})
+      padding-box,
+    linear-gradient(to right, #4673fa, darkorchid) border-box;
```

Comment:
lul first time seeing "darkorchid"

Reply:
Good catch - changing to match the button gradient :)